samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -18,22 +22,75 @@
 #error SW0_GPIO_NAME or SW0_GPIO_CONTROLLER needs to be set
 #endif
 #endif
-#define PORT SW0_GPIO_CONTROLLER
+#define PORT0 SW0_GPIO_CONTROLLER
 
 /* change this to use another GPIO pin */
 #ifdef SW0_GPIO_PIN
-#define PIN     SW0_GPIO_PIN
+#define PIN0     SW0_GPIO_PIN
 #else
 #error SW0_GPIO_PIN needs to be set
 #endif
 
 /* The switch pin pull-up/down flags */
 #ifdef SW0_GPIO_FLAGS
-#define PIN_FLAGS SW0_GPIO_FLAGS
+#define PIN0_FLAGS SW0_GPIO_FLAGS
 #else
 #error SW0_GPIO_FLAGS needs to be set
 #endif
 
+/* If second button exists, use it as right-click. */
+#ifdef SW1_GPIO_PIN
+#define PIN1	SW1_GPIO_PIN
+#endif
+
+#ifdef SW1_GPIO_NAME
```

Comment:
Actually, the first version I wrote, used this marco to detect if a PIN was defined or not. But in that times Zephyr doesn't support write BUTTON node in dts, somethings like SW0_GPIO_CONTROLLER are defined in board.h

But Zephyr support adding BUTTON node in dts now, so this check is useless. I think somethings like https://github.com/zephyrproject-rtos/zephyr/blob/93fc23bc7e458f9155c59980a86226a00b69aa9a/samples/basic/blinky/src/main.c#L11 is enough.

@galak Could you review this sections?

Reply:
It seems that we can simplify it and use only  SW*_GPIO_CONTROLLER, SW*_GPIO_FLAGS, SW*_GPIO_PIN now

Reply:
Removed SW*_GPIO_NAME ifdefs.